prov/sockets/src/sock_rma.c

```diff
@@ -262,7 +266,7 @@ ssize_t sock_ep_rma_writemsg(struct fid_ep *ep, const struct fi_msg_rma *msg,
 
 	SOCK_EP_SET_TX_OP_FLAGS(flags);
 	if (flags & SOCK_USE_OP_FLAGS)
-		flags |= tx_ctx->attr.op_flags;
+		flags |= op_flags;
```

Comment:
> > @@ -262,7 +266,7 @@ ssize_t sock_ep_rma_writemsg(struct fid_ep *ep, const
> struct fi_msg_rma *msg,
> >
> >  	SOCK_EP_SET_TX_OP_FLAGS(flags);
> >  	if (flags & SOCK_USE_OP_FLAGS)
> > -		flags |= tx_ctx->attr.op_flags;
> > +		flags |= op_flags;
> 
> We need to set the tx/rx flags in alias and setopt. That will take care of
> this.

The op_flags for an alias endpoint separate from the op_flags for the base endpoint.  One cannot change the other.